### ISSUE TITLE
Timer

### ISSUE THREAD

**Issue by freecicero** [cliambrown/EasyNoise#1]
For sleep use, perhaps add a timer?

**Comment by cliambrown** [comment]
That's a great idea! Unfortunately, I'm not sure I'll have time to add a timer in the near future.

**Comment by gbdomubpkm** [comment]
Absolutely essential indeed for this great app to take on its full meaning.

**Comment by feinmann** [comment]
Volume fade-out timer would be awesome.

**Comment by cliambrown** [comment]
Thanks for all of the feedback. I'll work on this feature for a future version.

I'll leave this thread open for specific requests related to this feature — if you just want to support the idea, please leave an emoji reaction on the top comment rather than adding a new comment.

**Comment by ltguillaume** [comment]
Thanks for the great work on this app!

For what it's worth, I just created a web app for this, primarily to have a timer included (and to test the Web Audio API). See https://codeberg.org/ltguillaume/soothy.

Can be tested directly at https://ltguillaume.codeberg.page/soothy

It's using the same samples as the ones collected by @cliambrown for this app (see credits).

**Comment by OnyxJet** [comment]
Yeah a timer feature that would shut off the app at a set time word be convenient.
